### PR TITLE
docs:Added small page with devel setup guide

### PR DESCRIPTION
**Pull request by david-caro**
Change-Id: I5382513d53719e18ef22e7b7617c2e30b5b6b6e5
Signed-off-by: David Caro <dcaroest@redhat.com>

### CODE REVIEW

**Comment by david-caro** [comment]
ok to test

**Comment by david-caro** [comment]
test this please

**Comment by david-caro** [comment]
test this please

**Comment by david-caro** [comment]
retest this please

**Comment by david-caro** [comment]
retest this please

**Comment by david-caro** [comment]
retest this please

**Comment by david-caro** [comment]
test this please

**Comment by david-caro** [comment]
ok to test

**Comment by david-caro** [comment]
test this please

**Comment by david-caro** [comment]
test this please

**Comment by david-caro** [comment]
ok to test

**Comment by david-caro** [comment]
test this please

**Comment by david-caro** [comment]
test this please

**Comment by david-caro** [comment]
test this please

**Comment by david-caro** [comment]
test this please

**Comment by david-caro** [comment]
Fixes #1

File: docs/Dev_env_setup.rst [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+Here are some guidelines on how to set up your development environment for the 
+lago project.
+
+
+Requirements
+-------------
+
+You'll need some extra packages to get started with the code for lago, assuming
+you are runnig fedora::
```
Comment (mykaul) [inline docs/Dev_env_setup.rst]: Fedora

File: docs/Dev_env_setup.rst [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+  > sudo dnf install git mock libvirt-daemon qemu-kvm autotools
+
+And you'll need also a few python libs, you can install them from the repos or
```
Comment (mykaul) [inline docs/Dev_env_setup.rst]: python -> Python
'you can install them from' -> 'which you you can install from'

File: docs/Dev_env_setup.rst [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+use venv or similar, for the sake of this example we will use the repos ones::
+
+  > sudo dnf install python-flake8 python-nose python-dulwich yapf
+
+Yapf is not available on older fedoras or CentOS, you can get it from the
+`official yapf repo`_ or try on `copr`_.
+
+Now you are ready to get the code::
+
+  > git clone git://gerrit.ovirt.org/lago
```
Comment (mykaul) [inline docs/Dev_env_setup.rst]: Github?

File: docs/Dev_env_setup.rst [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+
+From now on all the commands will be based from the root of the cloned repo::
+
+  > cd lago
+
+
+Style formatting
+------------------
+
+We will not accept any patch that does not pass pep8 and that it's not formatted
```
Comment (mykaul) [inline docs/Dev_env_setup.rst]: Change to a positive tone: We accept patches that pass PEP8 and YAPF formatting only.

**Comment by david-caro** [comment]
ci merge please

**Comment by david-caro** [comment]
ci please merge

There was a connectivity issue on jenkins

**Comment by david-caro** [comment]
ci merge please

**Pull request merged**